### ISSUE TITLE
Anomaly events spawn in area that do not exist on Box

### ISSUE THREAD

**Issue by Denthamos** [ParadiseSS13/Paradise#11943]
**Problem Description**:
Events with anomalies can spawn in areas of the map that do not exist on box station.

**What did you expect to happen**: 
Anomalies should spawn in areas that exist on the box map, and not in areas that are not present anymore/present on other maps.

**What happened instead**:
Anomalies are spawning in areas that players cannot reach.

**Why is this bad/What are the consequences:**
It prevents the anomaly from doing its part as an event in the round and the crew cannot interact with it in order to get anomaly cores.

**Steps to reproduce the problem**:
Not sure how you can reproduce it given its RNG spawn location for the event.

**When did the problem start happening**:
Possibly started during the mapping Z level redesigns a year ago or so.

![anomaly](https://user-images.githubusercontent.com/24213815/62065966-bf885e80-b1f5-11e9-8bc5-efb0e0e2ffcf.PNG)

**Comment by florkeger** [comment]
Given the fact I looked at the code - while not being too sure on how the selector works - I'd assume this happens due to areas being referenced on our map that actually don't exist anymore.

For example: I found residue of the Fitness Room area inside the current Holodeck, so I guess that's why it'd call it there since it found this particular place and chose it as it still existed. Not sure if that's how it works however. If it does, nice, if not, someone else has to take a look at it.
